software/viewer/src/components/map/MapScene3D.tsx

```
@@ -134,8 +111,8 @@ export const MapScene3D = forwardRef<MapScene3DHandle, MapScene3DProps>(
             controls.setLookAt(0, 600, 0, 0, 0, 0, true);
             break;
           case 'tracking':
-            // Track first active drone
-            const activeDrone = mockDrones.find(d => d.status === 'active');
+            // Track first active telemetry drone.
+            const activeDrone = telemetryDrones.find(d => d.status === 'active');
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)